### DISCUSSION TITLE
Wayland multiple monitor resetting positions (with semi-solution)

### DISCUSSION THREAD

**Discussion by horvay** [basecamp/omakub#380] [Show and tell]
If this isn't the right place to post this, I'll remove it, but I thought it may help others with the same problem:
---
If anyone has the issue with Wayland where it keeps resetting your monitor setup, let me know if you find a good solution. 

For now, I did this which works pretty well:

Install arandr `sudo apt install arandr`

Run arandr, and set up your monitor layout, and then save the script (seen in the screenshot). This creates an .sh file.

![GaIySoaWIAAZl5H](https://github.com/user-attachments/assets/f0db21f4-32fb-400a-b209-02640e6ec23b)

Then go to settings and set up a custom shortcut (see screenshots). I did Shift + Alt + 5

![GaIzGb7WIAAv0Lf](https://github.com/user-attachments/assets/4b0b86aa-95b6-4599-a45f-c281a329ca8d)
![GaI01A-WMAEmfXC](https://github.com/user-attachments/assets/af39e20b-27ea-4c72-8fb3-6cc54e93e982)

Now whenever it messes up the monitors, I just hit that shortcut and everything is fixed. 🥂

**Comment by jaykilleen** [comment]
I am getting this issue, although I did not know it. I'll try what you have documented. It's a pain.
